topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.md

```diff
@@ -485,4 +516,46 @@ The next example is *SRR12416842* from an Indonesia [study](https://www.microbio
 
 There is something clearly wrong with sample SRR12416842, perhaps indicating sample contamination. This example of a sample that doesn't map to the reference genome illustrates that even when sequence quality is good, sequence data problems can become apparent in later steps of analysis and it is important to always have a sense of what results to expect. You can develop a better sense of what quality control results to expect by first practicing techniques with known data before analysing new samples.
 
-We hope you enjoyed this tutorial!
+# Processing many samples at once: Collections and Workflows (optional)
+
+In the tutorial thus far we have focused on processing single samples, where two read datasets (forward and reverse reads) are associated with a single sample. In practice, sequence analysis typically involves analysing batches of samples and we run the same
+analysis steps for each sample in the batch. Galaxy supports working with batches using collections and workflows. 
+
+If you are new to collections, you can learn more about them in [this tutorial]({% link topics/galaxy-interface/tutorials/collections/tutorial.md %}).
+
+If you have followed all of the steps of this tutorial, you will have two samples, one named *004-2* and another *SRR12416842*. Create a collection of lists of pairs from these samples and call the collection `samples`.
+
+You can now run the **TB Variant Analysis**, launching it directly from WorkflowHub.eu on several of the public Galaxy servers.
+
+{% snippet faqs/galaxy/workflows_run_wfh.md title="TB Variant Analysis v1.0" wfhub_id="1035" %}
+
+> <hands-on-title>Analysing samples with the TB Variant Reporting workflow</hands-on-title>
+>
+> 1. **Run the TB Variant Analysis workflow** {% icon workflow %} using the following parameters
```

Comment:
For running the WF you could also use explanation like in https://training.galaxyproject.org/training-material/topics/variant-analysis/tutorials/sars-cov-2-variant-discovery/tutorial.html#hands-on-import-the-workflow-for-your-data-into-galaxy

Reply:
ah missed the snippet suggested by @hexylena that you put in already. That's even better.

Reply:
The snippet I included allows people to run the workflow directly on any of the usegalaxy.* servers. There are also FAQs for importing the workflow: `{% snippet faqs/galaxy/workflows_import.md %}` but is this still needed with the snippet I included?

Reply:
no it's fine with just this snippet, that will let them run it on any recent galaxy they can access.

 sorry we didn't document the non-wfh version well enough and you had to upload it. I'll fix that documentation this week.